### PR TITLE
Admin Main Page Update

### PR DESCRIPTION
**Pull request by ohamuy**
This is an update to the admin page, customizing it and altering it to make it cohesive with terrastories. This simply adds cosmetic changes to the UI, no functionality change.

### CODE REVIEW

**Comment by ohamuy** [comment]
Before: 
Screen Shot 2020-07-25 at 11 47 51 AM

After:
Screen Shot 2020-07-25 at 11 54 57 AM

**Comment by ohamuy** [comment]
Before:
Screen Shot 2020-07-25 at 11 47 51 AM

After:
Screen Shot 2020-07-31 at 10 41 46 AM

File: rails/app/assets/stylesheets/admin/_main_content_style.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+.import-icon {
+    margin-right: 5px;
+}
+
+.import-file-btn:first-child {
+    border-radius: 10px;
+    background-color: red;
+}
+
+//This is code that is partially working
```
Comment (FeminismIsAwesome) [inline rails/app/assets/stylesheets/admin/_main_content_style.scss:61]: can we remove these comments, maybe add an issue for the thing that still needs to be addressed?

pet peeve: commented code haha

Comment (ohamuy) [reply rails/app/assets/stylesheets/admin/_main_content_style.scss]: Done!

**Pull request merged**